tpl/bootstrap.php

```diff
@@ -524,18 +524,18 @@
 				</div>
 			</footer>
 		</main>
-		<div id="serverdata" class="hidden" aria-hidden="true">
 <?php
 if ($DISCUSSION):
 ?>
+		<div id="serverdata" class="hidden" aria-hidden="true">
```

Comment:
Oh, yeah good idea. Basically it seems the "serverdata" is actually no data anymore, but just the comment templates.

So we could remove the whole if thing and get that part static, too. But taht's a thing to do later…